source/Keystone/ApplicationCredentials.rst

```diff
@@ -0,0 +1,179 @@
+=======================
+Application Credentials
+=======================
+
+Application credentials can be used to allow applications or other users certain access to specific projects
+without embedding the main user's password in the application configuration.
+
+Application credentials can be created either through the web interface or command line
+
```

Comment:
It would be good to link our user credentials as pre-requisite knowledge. Then we can write something along the lines of `we assume the user is familiar with clouds.yaml usage` rather than duplicate docs